lte/gateway/c/core/oai/tasks/amf/amf_app_ue_context.cpp

```diff
@@ -217,6 +218,7 @@ ue_m5gmm_context_s* amf_ue_context_exists_amf_ue_ngap_id(
 smf_context_t* amf_insert_smf_context(
     ue_m5gmm_context_s* ue_context, uint8_t pdu_session_id) {
   smf_context_t smf_context;
+  memset(&smf_context, 0, sizeof(smf_context));
```

Comment:
add unit test coverage for `amf_insert_amf_context`

Reply:
comment addressed added test coverage.